lib/community/widgets/post_card.dart

```diff
@@ -29,6 +29,7 @@ class PostCard extends StatelessWidget {
     final bool showFullHeightImages = context.read<ThunderBloc>().state.preferences?.getBool('setting_general_show_full_height_images') ?? false;
     final bool showVoteActions = context.read<ThunderBloc>().state.preferences?.getBool('setting_general_show_vote_actions') ?? true;
     final bool showSaveAction = context.read<ThunderBloc>().state.preferences?.getBool('setting_general_show_save_action') ?? true;
+    final bool hideNsfwPreviews = context.read<ThunderBloc>().state.preferences?.getBool('setting_general_show_save_action') ?? true;
```

Comment:
noticed this is using a different setting than the one set in the settings - it should be `setting_general_hide_nsfw_previews`

Reply:
My bad!
Will fix.